spyder/plugins/projects/tests/test_plugin.py

```diff
@@ -25,16 +25,25 @@
 from spyder.py3compat import to_text_string
 
 
-# =============================================================================
-# Fixtures
-# =============================================================================
+# ---- Fixtures
```

Comment:
??? I thought this was our standard test file formatting for new and substantially revised files? I've never seen the ``# ---- NAME`` formatting anywhere else in our code (or anywhere else that I've seen) while it seems the previous is much more conventional (and besides, looks way better and clearer IMO).

Reply:
I agree with @CAM-Gerlach here. Please revert.

Reply:
!!! The reason I did this was because special comments are added to the Outline Explorer, which I find very useful to separate at a glance the fixtures from the tests. But that's ok, I'm going to revert it.

![image](https://user-images.githubusercontent.com/10170372/49959781-8efcea00-fedc-11e8-9600-32a15370f6b0.png)

Reply:
Sorry, then I agree with your change. Please remove your last commit.

Reply:
What about

```python
# =============================================================================
# ---- Fixtures
# =============================================================================
```

That keeps most of the existing convention (and more visible/better looking styling) and only requires adding four characters on one line to existing files as they are modified, and is more visible inline without the outline explorer while still achieving the same affect with it.

Reply:
👍 👍 👍 I think this is a good idea. If @ccordoba12 agrees with it I will do that.

Reply:
Sure, I agree with that too.